src/declarativedbusinterface.cpp

```diff
@@ -413,6 +413,103 @@ void DeclarativeDBusInterface::componentComplete()
     connectSignalHandler();
 }
 
+static QVariant unwind(const QVariant &val)
+{
+    QVariant res;
+
+    int type = val.userType();
+
+    if( type == val.type() ) {
```

Comment:
Code formatting differs from rest of the file/project: leave a space after reserved words, start else stmts on the same line that contains the curly brace closing the preceding if.